### PR TITLE
My portfolio

### PR DESCRIPTION
**Pull request by RaphDasilva**
# I Created my portfolio site using the following 
- CSS.
- HTML
- created .gitignore file, and ignored the following
- - node_modules
- created a readme.md file to explain my project.
- made use of linters to check for errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <nav>
```
Comment (nedjwab) [inline index.html:11]: [ ] the section's container should be a `header` tag,[Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header).kindly fix that🤓.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        <div id="nav-container">
+            <div class="nav-item">
```
Comment (nedjwab) [inline index.html:13]: [ ] the logo should have a link (`a` tag). This is for three reasons: convention, semantics, and user experience(UX).[Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. kindly fix that 🤓

File: index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                <h1>Welcome</h1>
+            </div>
+            <div class="nav-item nav-item2"><img src="img/Icon - Menu.png" alt="Menu Icon"></div>
+        </div>
+    </nav>
+
+    <section>
+        <div id="home">
+            <div class="welcome-text first-welcome-text">
+                <p>Hey there. I’m Raphael
```
Comment (nedjwab) [inline index.html]: [ ] this is the main title of your website, so it should be an `h1` tag, we use the `h1` tag If we want that google (and other search engines) know about a specific topic. kindly fix that 🤓.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+                    <div id="color-green">
```
Comment (nedjwab) [inline index.html]: [ ] we use a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`, This way will be semantically correct and the screen readers and other technologies will thank you.kindly fix that 🤓

File: index.html [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+                        I’m a software developer</div></p>
+            </div>
+            <div class="welcome-text second-welcome-text">
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+            </div>
+            <div id="social-icons">
```
Comment (nedjwab) [inline index.html]: [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. kindly fix that 🤓

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @RaphDasilva  

Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the project requirements, but you still have some issues: 
 
check the comments below 👇👇
 
## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @RaphDasilva 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - Good implementation of some of the project requirements ✔
 - GitHub flow was correctly used ✔.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**